src/app/features/tech-record/components/tech-record-amend-vrm/tech-record-amend-vrm.component.html

```diff
@@ -20,19 +20,60 @@ <h2 class="govuk-heading-l">{{ makeAndModel }}</h2>
     </div>
   </dl>
 
-  <form [formGroup]="form">
-    <app-text-input #newVrm appToUppercase appNoSpace appTrimWhitespace formControlName="newVrm" [width]="width"></app-text-input>
-
-    <app-radio-group formControlName="isCherishedTransfer" [options]="reasons"></app-radio-group>
-
-    <div *ngIf="newVrm.value.toUpperCase() !== techRecord.primaryVrm && newVrm.value.length">
-      <p class="govuk-heading-l">Are you sure you want to change this VRM?</p>
-      <p class="govuk-body">Amending this VRM will also amend the VRM for all other tech records present.</p>
-    </div>
-
-    <app-button-group>
-      <app-button id="submit-vrm" (clicked)="handleSubmit()">Save</app-button>
-      <app-button id="cancel-amend-vrm" design="link" data-module="govuk-button" (clicked)="navigateBack()">Back</app-button>
-    </app-button-group>
-  </form>
+  <ng-container [ngSwitch]="isCherishedTransfer">
```

Comment:
instead of using a `switchCase` for a boolean, I would recommend using `*ngIf="isCherisherTransfer; else correctingAnError"` using `<ng-template #correctingAnError>`